App.js

```diff
@@ -62,7 +62,7 @@ class App extends Component {
       {
         storage: AsyncStorage,
         transforms: [encryptor],
-        blacklist: ['user'],
+        blacklist: ['entities', 'pagination', 'errorMessage', 'user'],
```

Comment:
With this and after some thought, I'm really not too concerned with adding more and more normalized entities to our store. If the data isn't persisting, I really don't think this would be too much of an issue 👍